modules/multiplayer/editor/replication_editor.cpp

```diff
@@ -399,6 +396,11 @@ void ReplicationEditor::_add_pressed() {
 		np_text = "." + np_text;
 	}
 	NodePath path = NodePath(np_text);
+	if (path.is_empty()) {
+		error_dialog->set_text(vformat(TTR("Invalid property path: '%s'"), np_text));
+		error_dialog->popup_centered();
+		return;
+	}
```

Comment:
This is unrelated to this specific PR, but should we use `EditorNode::get_singleton()->show_warning` instead of a custom dialog?

Reply:
Well, we are using that error dialog in a bunch of other places, including the recently merged https://github.com/godotengine/godot/pull/65558 .
Should I make a pass to replace all of them in this PR or do it in a separate one?

Reply:
It's definitely for a separate PR, would be great if you did a pass!

Reply:
Well, I ended up attaching it to this PR, sorry about that :sweat_smile: ... it was smaller than expected and really straight forward: https://github.com/faless/godot/commit/fb67517edc92d73ddee3a1ef0d5c064be123a0ad .
(it also highlighted a small issue in the current PR where it was trying to `set_checked` instead of `set_range` with more than 64 on change vars)